### DISCUSSION TITLE
How disable auto start on boot?

### DISCUSSION THREAD

**Discussion by meirroth** [KRTirtho/spotube#1284] [Q&A] [answered]
App launches on windows computer start. How to disable that?

**Comment by KRTirtho** [comment]
There's 1M+ article written about it lol
https://pureinfotech.com/disable-startup-apps-windows-11/#:~:text=To%20disable%20apps%20at%20startup,Startup”%20folder,%20or%20Registry.

**Comment by KRTirtho** [comment, marked as answer]
By default Spotube doesn't auto start and there's no option in the app itself. Usually the installer have that option. Which is also turned off by default.

**Comment by meirroth** [comment]
@KRTirtho Got it. Thanks! Great app btw 😀
